config-overrides.js

```diff
@@ -2,7 +2,19 @@ const webpack = require('webpack')
 
 module.exports = function override(config) {
   const fallback = config.resolve.fallback || {}
-  Object.assign(fallback, {})
+  require('react-app-rewire-postcss')(config /*, options */)
+
+  Object.assign(fallback, {
+    crypto: require.resolve('crypto-browserify'),
```

Comment:
I went through each error and added the polyfill the package error message requested/said it needed/depended on.

Reply:
@benschac amaaaazing. This part is super informative. Speaks to the power of rewire.